crates/core/src/partials/mod.rs

```diff
@@ -1,3 +1,36 @@
+//! # Using partials
+//! To use `{% include %}` or `{% render %}` tags in a template, you first need to compile these
+//! included files as template partials.
+//!
+//! Example:
+//!
+//! ```liquid
+//! # common.liquid
+//! Number: {{ i }}
+//! ```
+//!
+//! ```rust,no_run,compile_fail
+//! // Build template partials using an eager, in-memory source compiler.
+//! // Other compilation policies also exist depending on specific needs.
+//! type Partials = EagerCompiler<InMemorySource>;
+//!
+//! let partials = {
+//!   let mut _partials = Partials::empty();
```

Comment:
Could you remove the `_` prefix?  That tells the compiler that it can be unused.  We can just call it `partials`

Reply:
Done.